.github/workflows/functional_verified.yml

```diff
@@ -81,9 +82,8 @@ jobs:
           fi
           sudo touch /var/run/job.in.progress
           rm -rf cleanup.sh install_cleanup.sh
-          # after this PR is merged, update URLs to get the scripts from github master
-          wget https://storage.googleapis.com/minikube-ci-utils/cleanup.sh
-          wget https://storage.googleapis.com/minikube-ci-utils/install_cleanup.sh
+          wget https://raw.githubusercontent.com/kubernetes/minikube/master/hack/gh_actions/cleanup.sh
+          wget https://raw.githubusercontent.com/kubernetes/minikube/master/hack/gh_actions/install_cleanup.sh
```

Comment:
Do we need to run the clean up too?

Reply:
We call `./install_cleanup.sh` below, which copies `cleanup.sh` into the hourly cron folder.